data/rule_data.yml

```diff
@@ -88,7 +88,14 @@ rule_data:
     - 'Red Hat, Inc.'
   - name: version
     description: Version of the image.
-
+  - name: org.opencontainers.image.created
+    description: >-
+      The creation timestamp of the image. This label must always be set by the Konflux build task for on-prem product releases. The policy must also support reproducible builds by allowing users to override the timestamp using the SOURCE_DATE_EPOCH environment variable.
+    effective_on: "2026-06-07T00:00:00Z"
+  - name: cpe
+    description: >-
+      The CPE (Common Platform Enumeration) identifier for the product, e.g., cpe:/a:redhat:openshift_gitops:1.16::el8. This label is required for on-prem product releases.
+    effective_on: "2026-06-07T00:00:00Z"
```

Comment:
The date is in the past already, so it's going to have no effect and might as well be omitted. Did you mean to set it to some time in the future?

Reply:
Oh sorry I didn't read it properly. 2026. Disregard!